packstream/packstream-specification-1.md

```diff
@@ -781,16 +781,21 @@ DateTime::Structure(
 )
 ```
 
-- The `seconds` are seconds since the adjusted [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)). This is not [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time).
 - The `tz_offset_seconds` specifies the offset in seconds from UTC.
+- The `seconds` are the above offset added to the seconds since the [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)).
 
-
-To convert to UTC:
-
+`1655472600` is the numbers of seconds since the Unix Epoch in `Fri Jun 17 2022 13:30:00 UTC`.
+If the time zone offset is +1 hour, i.e. 3600 seconds, then the corresponding `DateTime` instance is as follows:
 ```
-utc_nanoseconds = (seconds * 1000000000) + nanoseconds - (tx_offset_seconds * 1000000000)
+{
+  seconds: 1655476200 # that's the Unix timestamp (1655472600) + the timezone offset (3600)
+  nanoseconds: 0,
+  tz_offset_seconds: 3600
+}
```

Comment:
This sounds like it's the new format (Packstream V2). Or am I misreading this?

Reply:
The new format would be `1655472600`, not `1655476200=1655472600+3600`.